lib/compat/wordpress-6.2/block-editor-settings.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Adds settings to the block editor.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Adds styles and __experimentalFeatures to the block editor settings.
+ *
+ * @param array $settings Existing block editor settings.
+ *
+ * @return array New block editor settings.
+ */
+function gutenberg_get_block_editor_settings_6_2( $settings ) {
+	if (
+		defined( 'REST_REQUEST' ) &&
+		REST_REQUEST &&
+		isset( $_GET['context'] ) &&
+		'mobile' === $_GET['context']
+	) {
+		return $settings;
+	}
```

Comment:
Why do we need this check? Is there an issue with loading custom CSS on mobile?

Let's add some context with an inline comment; it will help when backporting changes to the core.

Reply:
some bad copy pasta on my part here without thinking, sorry for the confusion, have removed for now.